src/plugins/unified_search/public/saved_query_form/save_query_form.tsx

```diff
@@ -137,8 +143,7 @@ export function SaveQueryForm({
           defaultMessage: 'Name',
         })}
         helpText={i18n.translate('unifiedSearch.search.searchBar.savedQueryNameHelpText', {
-          defaultMessage:
-            'Name cannot contain a leading or trailing whitespace and must be unique.',
+          defaultMessage: 'Name cannot contain leading or trailing whitespace and must be unique.',
```

Comment:
```suggestion
          defaultMessage: 'Name must be unique.',
```

Does it make sense to have it if we trim the name anyway?

<img width="290" alt="Screenshot 2024-01-24 at 11 45 21" src="https://github.com/elastic/kibana/assets/1415710/24cc40ba-31eb-4f32-bdb9-a6273dbad3c1">

Reply:
I think we're good to remove this help text.

Reply:
Sounds good! Removed here: https://github.com/elastic/kibana/pull/170599/commits/303e1b85473def3dad16db6f211108a8c75439e0.